src/components/error/GlobalErrorOverlay.react.js

```diff
@@ -14,14 +15,10 @@ export default class GlobalErrorOverlay extends Component {
 
         let frontEndErrors;
         if (toastsEnabled) {
-            let errors = [];
-            if (error.frontEnd.length) {
-                errors = error.frontEnd;
-            }
-
-            error.backEnd.forEach(backEndError => {
-                errors.push(backEndError);
-            });
+            const errors = concat(
+                error.frontEnd,
+                error.backEnd
+            );
```

Comment:
@byronz - this should fix 
![image](https://user-images.githubusercontent.com/1280389/56067956-0c315200-5d4b-11e9-99c0-040eb3ab6573.png)

Reply:
good catch ! :+1: